azure-iot-device/azure/iot/device/provisioning/pipeline/exceptions.py

```diff
@@ -0,0 +1,20 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module defines an exception surface, exposed as part of the pipeline API"""
+
+# For now, present relevant transport errors as part of the Pipeline API surface
+# so that they do not have to be duplicated at this layer.
+from azure.iot.device.common.pipeline.pipeline_exceptions import *
+from azure.iot.device.common.transport_exceptions import (
+    ConnectionFailedError,
+    ConnectionDroppedError,
+    # TODO: UnauthorizedError (the one from transport) should probably not surface out of
```

Comment:
Nit: good explanation, but please add your initials, like "OKTODO" so we know who wrote it when we come across it in the future.